manage-postgresql-user.py

```diff
@@ -0,0 +1,543 @@
+#!/usr/bin/python
+import logging
+import os
+import psycopg2
+import boto3
+import random
+import string
+import json
+from datetime import datetime
+
+libraries = ("boto3", "psycopg2")
+
+# Initialise logging
+logger = logging.getLogger(__name__)
+log_level = os.environ["LOG_LEVEL"] if "LOG_LEVEL" in os.environ else "INFO"
+logger.setLevel(logging.getLevelName(log_level.upper()))
+logger.info("Logging at {} level".format(log_level.upper()))
+
+# Use appropriate CA SSL cert to verify RDS identity. Use "AmazonRootCA1.pem" for Aurora Serverless.
+# Defaults to "rds-ca-2019-2015-root.pem" to suit normal RDS.
+rds_ca_cert = (
+    os.environ["RDS_CA_CERT"]
+    if "RDS_CA_CERT" in os.environ
+    else "/var/task/rds-ca-2019-2015-root.pem"
+)
+
+
+def generate_password():
+    """Generate a password.
+
+    Args:
+        None
+
+    Returns:
+        str: generated password
+    """
+    valid_chars = string.ascii_letters + string.digits + string.punctuation
+    invalid_chars = ["/", "@", '"', "\\", "'"]  # Not allowed in a MySQL password
```

Comment:
can you check the invalid characters for Postgres and update teh comment at the end